loadable_apps/loadable_sample/micomapp/Makefile

```diff
@@ -85,58 +72,35 @@ KERNEL_VER = 2.0
 STACKSIZE = 4096
 PRIORITY = 220
 
-ifeq ($(CONFIG_COMPRESSED_BINARY),y)
-COMPRESSION_TYPE = $(CONFIG_COMPRESSION_TYPE)
-BLOCK_SIZE = $(CONFIG_COMPRESSION_BLOCK_SIZE)
-else
-COMPRESSION_TYPE = 0
-BLOCK_SIZE = 0
-endif
+include $(TOPDIR)/../loadable_apps/loadable.mk
 
-APPDEFINE = ${shell $(TOPDIR)/tools/define.sh "$(CC)" __APP_BUILD__}
+###################################
+# Cases of Flat and Protected build
+# Merge into libapps.a
+###################################
+else
```

Comment:
I am wondering if this is correct? Do we need to merge these loadable apps into libapps?

Reply:
This is an else case of app separation. Without app separation, this is also a part of app so that I think this is right.